tmt/steps/provision/mrack.py

```diff
@@ -567,6 +567,23 @@ def _transform_system_numa_nodes(
         children=[MrackHWBinOp('numanodes', beaker_operator, actual_value)])
 
 
+def _transform_system_model_name(
+        constraint: tmt.hardware.NumberConstraint,
```

Comment:
Shouldn't it be `TextConstraint`?
```suggestion
        constraint: tmt.hardware.TextConstraint,
```